brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1261 Fix: Team - Line height not applying on tablet & mobile devices

### Description
Team: Title, description and designation line-height is not working for tablet and mobile devices on backend

### Screenshots
Issue - https://d.pr/v/h3Ku6z
Fix - https://share.getcloudapp.com/o0uGRQ0W

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
- Add a team block
- Go to style setting
- Set description line-height for tablet and mobile
- Check on backend

### Checklist:
